default.json

```diff
@@ -5,18 +5,27 @@
       "customType": "regex",
       "fileMatch": ["dprint.json", "dprint.jsonc", ".dprint.json", ".dprint.jsonc"],
       "matchStrings": [
-        "\"https://plugins.dprint.dev/(?<depName>[^/]+)-(?<currentValue>[^/]+?).wasm\""
+        "\"https://plugins.dprint.dev/(?<pluginName>[^/-]+)-(?<currentValue>[^/-]+?).wasm\""
```

Comment:
depName might having special role, I should check https://github.com/kachick/renovate-config-dprint/issues/44 after merging this PR